c7n/query.py

```diff
@@ -816,6 +816,9 @@ class TypeInfo(metaclass=TypeMeta):
         passed as this value. Further customizations of dimensions require subclass metrics filter
     :param cfn_type: AWS Cloudformation type
     :param config_type: AWS Config Service resource type name
+    :param config_id: Resource attribute that maps to the resourceId field in AWS Config. Intended
```

Comment:
can we add a default value here for config_id = None to the class.

Reply:
Thanks, good call :+1: